r_code/ts_feature_analysis.R

```diff
@@ -209,15 +183,31 @@ FeatureDensity <- function(data_full,id_var = "entry",
                          dplyr::distinct(!!as.name(id_var), .keep_all = T) %>%
                          dplyr::pull(Treated)))
   
-  feature_density <- ggplot2::ggplot(feature_treat_df)+
+  feature_density <- ggplot2::ggplot(feature_treat_df) +
     ggplot2::geom_density(aes(x=!!as.name(feature), color=Treated),alpha=0.4) +
     ggplot2::theme(panel.grid.major = element_blank(), 
                    panel.grid.minor = element_blank(),
                    panel.background = element_blank(), 
-                   axis.line = element_line(colour = "black"))+
-    ggplot2::scale_color_manual(values=c("#E69F00", "#56B4E9"))+
+                   axis.line = element_line(colour = "black")) +
+    ggplot2::scale_color_manual(values=c("#E69F00", "#56B4E9")) +
     ggplot2::labs(title = paste(feature, "Density by Treatment"))
   
   return(feature_density)
   
 }
+
+#' Identify and tag the treated units by ID.
+#'
+#' @inheritParams TSFeaturesPCScatterplot
+#'
+#' @return The original tibble of data, with a new Treated indicator column. 
+.IdentifyTreated <- function(data_full, id_var, treat_indicator){
+  # Identify treated entries.
+  treated_entries <- (data_full %>%
+                        dplyr::filter(!!as.name(treat_indicator) > 0) %>%
+                        dplyr::distinct(!!as.name(id_var)) %>% 
+                        dplyr::pull())
```

Comment:
```suggestion
                      dplyr::filter(!!as.name(treat_indicator) > 0) %>%
                      dplyr::distinct(!!as.name(id_var)) %>% 
                      dplyr::pull())
```

Reply:
I think the parenthesis at the beginning and end were messing up the default alignment for piping. It's a different change than requested, but it's what the other lines/files have used.